src/test/shared/vscode/env.test.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import assert from 'assert'
+import { getServiceEnvVarConfig } from '../../../shared/vscode/env'
+
+describe('getServiceEnvVarConfig', function () {
+    const envVars: string[] = []
+
+    afterEach(() => {
+        envVars.forEach(v => delete process.env[v])
```

Comment:
This assumes that the env var did not already exist. `__CODECATALYST_ENDPOINT` for example would be set by some pipelines that run the E2E tests. This test happens to not be an E2E test, so not an immediate problem.